<launch>

    <node name="ltm_json_loader" type="json_loader.py" pkg="ltm" output="screen">
        <param name="filename" type="str" value="$(find ltm)/test/_structure.json"/>
    </node>

</launch>